app.py

```diff
@@ -128,12 +128,15 @@
 PUNCTRE_LABEL = re.compile(r'.*-p$')
 INITIAL_PUNCT_LABELS = {'LRB-p', '[-p', '{-p'}
 
-def is_possible_punct_sequence(token):
-	return re.match(PUNCTRE, token) or token in [e['ptree_token'] for e in PUNCT_ESCAPING]
+def is_punct_postag(tag):
+	return tag in PUNCT_TAGS.values() or tag == SYMBOL_TAG
 
 def is_punct_label(label):
 	return re.match(PUNCTRE_LABEL, label) or label in [e['ptree_label'] for e in PUNCT_ESCAPING]
```

Comment:
Trying to figure out if `is_punct_label()` still makes sense as implemented. The first part merely checks if it is a string ending in `-p`, right? Is the second condition adding anything? Should invalid combinations like `Noun-p` be ruled out?

Reply:
If it is merely checking for the `-p` ending I would probably remove the helper function/regex altogether and just use `label.endswith('-p')`.

Reply:
You're right, this can be simplified -- see https://github.com/nschneid/activedop/pull/59/commits/3474da14e3cd59d5dda1aa999df54a68883452f2